csharp/extractor/Semmle.Extraction/Extractor/StandaloneExtractor.cs

```diff
@@ -14,9 +13,11 @@ public class StandaloneExtractor : Extractor
         /// <param name="pathTransformer">The object used for path transformations.</param>
         public StandaloneExtractor(ILogger logger, PathTransformer pathTransformer, CommonOptions options) : base(logger, pathTransformer)
         {
-            mode = ExtractorMode.Standalone;
+            Mode = ExtractorMode.Standalone;
             if (options.QlTest)
-                mode |= ExtractorMode.QlTest;
+            {
```

Comment:
The comment related to the use of { } was merely a question. I do see the value in not using { }, when the formatting we have chosen puts all { } on new lines (not sure if this is purely bike-shedding). I just wanted to know, if we have a strict C# coding standard for one way or the other.

Reply:
@tamasvajk has set up some [automated coding standard rules](https://github.com/github/codeql/blob/main/csharp/.editorconfig), which I think should take effect on format (I use format on save by default). So perhaps the rules do not require the braces.

Reply:
I prefer braces everywhere, but I didn't want to enforce my preferences onto the team. Instead I took the `.editorconfig` from the .NET docs site and the roslyn repo.